src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -22,7 +25,8 @@ object Ledger extends Logger {
       log.debug(s"All txs from block ${block.header} were executed")
 
       val worldToPersist = payBlockReward(Config.Blockchain.BlockReward, block, resultingWorldStateProxy)
-      val afterExecutionBlockError = validateBlockAfterExecution(block, worldToPersist)
+      val receipts = Seq() //FIXME: This should be obtained from block execution (done in EC-139)
```

Comment:
does that mean that this PR should be merged after https://github.com/input-output-hk/etc-client/pull/123 ?